src/KeyValidatorTrait.php

```diff
@@ -23,6 +19,12 @@
      */
     protected function validateKey($key)
     {
+        $matched = preg_match('/^[A-Za-z0-9_.]{1,64}$/', $key);
+
+        if (1 !== $matched) {
+            throw new InvalidArgumentException('Can\'t validate the specified key');
```

Comment:
Also valid points. I'd say the tests should include strings of 64 characters (including multi-bite characters), but not reject strings over 64 characters.